subsys/bluetooth/mesh/light_temp_srv.c

```diff
@@ -208,6 +208,7 @@ static void lvl_delta_set(struct bt_mesh_lvl_srv *lvl_srv,
 			  const struct bt_mesh_lvl_delta_set *delta_set,
 			  struct bt_mesh_lvl_status *rsp)
 {
+	static uint16_t corrective_delta;
```

Comment:
What if 2 temperature servers instantiated on the device?

Reply:
Good point, should have thought of that my self. Well then I guess we need to create a param in the server instance.